lib/utility/_query.js

```diff
@@ -39,6 +39,9 @@ exports._query = function _query(query, data, options, callback, executor) {
 		// Use resultSet if 'useResultSet' option is set
 		queryOptions.resultSet = true;
 		queryOptions.prefetchRows = prefetchRows;
+	} else if (options && options.write) {
+		// Save changes to the db
+		queryOptions.saveChanges = true;
```

Comment:
Did you consider using [autoCommit](https://github.com/oracle/node-oracledb/blob/master/doc/api.md#-42431-autocommit)? That way you wouldn't need another wrapper function to actually perform the commit.